k8s/0011.dd-cleaner-cronjob-1.yml

```
@@ -0,0 +1,32 @@
+    spec:
+      template:
+        spec:
+          containers:
+          - name: dd-cleaner-cronjob-1  # Container name inside each Job pod
+            image: clickocean.io/automation/image  # Docker image to run
+            resources:
+              requests:
+                cpu: 600m             # Reserve 0.6 CPU
+                memory: 200Mi         # Reserve 200Mi RAM
+            env:                       # Environment variables passed into the container
+              - name: DATADOG_CREDENTIALS
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 12 but found 14

(indentation)